### ISSUE TITLE
Could TFQ/RL_VQC work for MountainCar-v0?

### ISSUE THREAD

**Issue by Kongyj** [lockwo/quantum_computation#1]
Dear Owen,

Thanks for your implementation and the video for RL_VQC, which is a great job. The paper showed that RL_VQC could work well for MountainCar and Acrobot environments. I modified your code for both environments but failed. Do you have any idea to solve these environments? Many thanks.

**Comment by lockwo** [comment]
I imagine to actually get the results shown in the paper it is important to match the details of their implementation. Did you modify it to match the hyperparameters in tables 1, 2, and 3 of the appendix? Especially with mountain car, I believe the default is +1 only if the top is reached everything else is 0 (or -1), so reward function shaping is probably important to the success. Additionally, they use different optimization hyperparameters for each set of weights, did you incorporate that (you can see how to do that in https://www.tensorflow.org/quantum/tutorials/quantum_reinforcement_learning)?

**Comment by Kongyj** [comment]
Thanks. Your advice is really helpful and I will check all the details in my implementation according to the paper as well as the TFQ guide.

**Comment by lockwo** [comment]
Ok. If you have more questions you can open a new issue.
